presto-product-tests/src/main/java/com/facebook/presto/tests/hive/TestHiveCoercion.java

```diff
@@ -386,4 +434,14 @@ private static QueryResult executeHiveQuery(String query)
     {
         return testContext().getDependency(QueryExecutor.class, "hive").executeQuery(query);
     }
+
+    private static Map<Object, Object> namedStruct(Object... objects)
```

Comment:
With `ImmutableMap.of(...)`, you don't need this helper (up to 5 key-value entries).

When you go over, we can't do this any more. But we wouldn't do it anyways because readability is pretty bad at that point anyways. Instead, we do:
```
ImmutableMap.<K, V>builder()
        .put(..., ...)
        .put(..., ...)
        .put(..., ...)
        .put(..., ...)
        .put(..., ...)
        .build()

Reply:
I initially come across `ImmutableMap` but then I realized I couldn't use it because I want to test `null` value but `ImmutableMap` don't allow `null`.